### PR TITLE
[SER-4017] Added estimated delivery date to response object.

### PR DESCRIPTION
**Pull request by alvinnguyenHL**

### CODE REVIEW

**Comment by alvinnguyenHL** [comment]
Looks like the tests are passing with php 5.6 so we should be good? Don't think they have ever passed with php 5.5 since I didn't touch any OnTrac files and thats one of the tests that failed. The code that uses the estimated delivery date is in symfony, this code just initializes it.

File: src/Provider/FedexProvider.php [hunk 1]
```diff
@@ -159,6 +159,11 @@ private function parseTrackReply($xml)
             );
         }
 
-        return new ShipmentInformation($events);
+        $estimatedDeliveryDate = null;
```
Comment (oumsofiane1) [inline src/Provider/FedexProvider.php]: this will raise an exception if TrackReply or TrackDetails are null

**Comment by oumsofiane1** [comment]
Tests are failing

**Comment by oumsofiane1** [comment]
@alvinnguyenHL if we are going to stop support for PHP 5.5 we'll need to document that in our changelog, have a new release and update Travis to not test for PHP 5.5.

**Comment by alvinnguyenHL** [comment]
Jira ticket - https://jira.hautelook.net/browse/SER-4410

**Pull request merged**